sys/include/net/gnrc/netdev2.h

```diff
@@ -73,6 +73,16 @@ typedef struct gnrc_netdev2 {
     gnrc_pktsnip_t * (*recv)(struct gnrc_netdev2 *dev);
 
     /**
+     * @brief Set the state of this device
+     */
+    int (*_set_netdev_state)(struct gnrc_netdev2 *dev, netopt_state_t devstate);
```

Comment:
Wait... this is just about the state? Why not have a getter/setter for all options?